go/vt/vtgate/engine/ordered_aggregate.go

```diff
@@ -476,6 +478,7 @@ func merge(
 			data, _ := proto.Marshal(vgtid)
 			val, _ := sqltypes.NewValue(sqltypes.VarBinary, data)
 			result[aggr.Col] = val
+		case AggregateRandom:
```

Comment:
this is correct, but also a little confusing. maybe add a comment explaining what's going on here?